### PR TITLE
Ajout landing page A+

### PR DESCRIPTION
**Pull request by jdauphant**
image

@Aurelie-J Il faudrait que tu : 
- [x] Fournisse l'image SVG success sans fond blanc (il y a une erreur sur l'image)

### CODE REVIEW

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by Aurelie-J** [comment]
[https://drive.google.com/file/d/1uYf34eOdurY3XwW9TDDm_qjjT9E_n0vM/view?usp=sharing](url)

**Comment by Aurelie-J** [comment]
- [ ] Possible de remplacer la couleurs des textes qui apparaissent en noir par le code color:#00234d (darker template betagouv) 

- [ ] mettre aussi avec le code couleur color:#00234d (darker) les flècheq qui apparaissent sous les titres ADMINISTRATION+ Qu'est-ce que c'est ? etc.

- [ ] passer à 16px le texte dans les box bleues : font-size: 16px;

- [ ] le sous-titre sous le logo en haut à gauche "LE SERVICE... URGENTS" augmenter la line-height: 1.3;

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by Aurelie-J** [comment]
- [ ] agrandir taille du texte + numéro de tél qui apparait sur le bandeau bleu support@aplus.beta.gouv.fr 01 84 80 88 12 

- [ ] ajouter de l'espace à gauche et à droite de 👍   "Objectif : coopérer pour résoudre les blocages complexes ou les demandes urgentes" et "Des agents dans chaque administration sont mobilisés pour répondre aux urgences sociales" : width: 80%;
margin: auto;

- [ ] améliorer le rendu du footer : )
image

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

File: app/views/home.scala.html [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+                        >
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            >
+                                <h2>
+                                    6
+                                </h2>
+                                <p>
+                                    Sur A+ vous pouvez solliciter plusieurs administrations sur
+                                    une même demande.
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                   
```

File: app/views/home.scala.html [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+                        class="mdl-card usage-card mdl-shadow--2dp aplus-color--blue mdl-cell mdl-cell--4-col"
+                        >
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            >
+                                <h2>
+                                    2
+                                </h2>
+                                <p>
+                                    Sur A+ vous gardez une trace des échanges tant que la
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                     Sur Administration+, vous échangez ensemble sur un fil de discussion dédié.
```

File: app/views/home.scala.html [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+                                    demande est en cours.
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                   
```

File: app/views/home.scala.html [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+                        class="mdl-card usage-card mdl-shadow--2dp aplus-color--blue mdl-cell mdl-cell--4-col"
+                        >
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            >
+                                <h2>
+                                    3
+                                </h2>
+                                <p>
+                                    Sur A+ vous êtes en conformité par rapport à la
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                    Sur Administration+, vous gardez une trace des échanges tant que
```

File: app/views/home.scala.html [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+                                    confidentialité des données.
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                    la demande est en cours.
```

File: app/views/home.scala.html [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+                        >
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            >
+                                <h2>
+                                    4
+                                </h2>
+                                <p>
+                                    Les référents sont mobilisés et répondent habituellement au
+                                    plus tard sous 4 jours .
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                      
```

File: app/views/home.scala.html [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+                                </p>
+                            </div>
+                        </div>
+                        <div
+                        class="mdl-card usage-card mdl-shadow--2dp aplus-color--blue mdl-cell mdl-cell--4-col"
+                        >
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            ><h2>5</h2>
+                                <p>L’équipe d'A+ est disponible pour vous aider.</p>
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                <p>Les demandes sont habituellement résolues sous 4 jours.</p>
```

File: app/views/home.scala.html [hunk 8]
```diff
@@ -0,0 +1,399 @@
...
+                        >
+                            <div
+                            class="mdl-cell mdl-cell--7-col mdl-cell--1-offset-desktop mdl-cell--12-col-mobile"
+                            >
+                                <h4 class="mdl-typography--font-medium">
+                                    Vous rencontrez des difficultés à vous connecter ?
+                                </h4>
+                                <h5 class="mdl-typography--font-regular">
+                                    Contactez-nous, notre équipe est disponible pour vous
+                                    renseigner et vous aider
```
Comment (jdauphant) [inline app/views/home.scala.html]:
```suggestion
                                    renseigner et vous aider.
```

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by Aurelie-J** [comment]
Le lien Nous contacter dans le header doit faire remonter (système d'ancre) le bandeau bleu sur lequel se trouve les coordonnées du support : )

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

File: app/views/home.scala.html [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+                            <div
+                            class="mdl-card__title mdl-card--expand mdl-color-text--white mdl-typography--text-center"
+                            >
+                                <h2>
+                                    1
+                                </h2>
+                                <p>
+                                    Sur Administration+, vous pouvez faire intervenir plusieurs administrations
+                                    sur une demande. 
+                                    discussion.
```
Comment (Aurelie-J) [inline app/views/home.scala.html]:
```suggestion
                                  
```

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           

Coverage increased per file
===========================
+ app/controllers/HomeController.scala  6
         

Coverage decreased per file
===========================
- app/actions/LoginAction.scala  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=5058611&bid=16650154)

**Pull request merged**